deploy/common/k8sbased.sh

```diff
@@ -327,8 +327,24 @@ function launch_central {
         helm lint "$unzip_dir/chart" -n stackrox
         helm lint "$unzip_dir/chart" -n stackrox "${helm_args[@]}"
       fi
-      helm upgrade --install -n stackrox stackrox-central-services "$unzip_dir/chart" \
+
+      local helm_chart="$unzip_dir/chart"
+
+      if [[ -n "${CENTRAL_CHART_DIR_OVERRIDE}" ]]; then
+        helm_chart="${CENTRAL_CHART_DIR_OVERRIDE}"
+      fi
+
+      if [[ -n "${CENTRAL_CHART_VERSION_OVERRIDE}" ]]; then
+        helm_args+=(
+          --version="${CENTRAL_CHART_VERSION_OVERRIDE}"
+        )
+      fi
```

Comment:
If you provide downloaded chart in `CENTRAL_CHART_DIR_OVERRIDE`, you don't need `CENTRAL_CHART_VERSION_OVERRIDE`.